modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/akula.dm

```diff
@@ -35,54 +49,114 @@
+	"Yet, despite their differences, all Agurkrral citizens swim freely in their kingdom's waters. Even the most controlling border princes, even those in the Old Principalities working the slave trade, know better than to openly erode a citizen's right to life, property, and speech. Any alien species can become an Agurkrral citizen, and even non-citizens enjoy the right to life, with executions outright banned. The aristocracy remains well-educated, even the edgerunner warlords of the New Principalities, and the Kingdom as a whole enjoys its status as a nation that's now a true rival to Sol. Larger, more populated, and better developed; though, having to 'integrate' Solarian technologies, goods, and peoples to fully succeed. The Azuleans are even known as an environmentally-focused people; although they hold no care for lands they cannot make use of, modern nobles are still in charge of maintaining the biosphere of lands they control, to allow their strangely engineered flora and fauna to thrive, and for the people to have healthy and clean waters to live in.")
 
 /datum/species/akula/randomize_features(mob/living/carbon/human/human_mob)
 	var/main_color
-	var/second_color
-	var/random = rand(1,5)
-	//Choose from a variety of sharkish colors, with a whiter secondary and tertiary
+	var/secondary_color
+	var/tertiary_color
+	var/random = rand(1,4)
 	switch(random)
 		if(1)
-			main_color = "#668899"
-			second_color = "#BBCCDD"
+			main_color = "#1CD3E5"
+			secondary_color = "#6AF1D6"
+			tertiary_color = "#CCF6E2"
 		if(2)
-			main_color = "#334455"
-			second_color = "#DDDDEE"
+			main_color = "#CF3565"
+			secondary_color = "#d93554"
+			tertiary_color = "#fbc2dd"
 		if(3)
-			main_color = "#445566"
-			second_color = "#DDDDEE"
+			main_color = "#FFC44D"
+			secondary_color = "#FFE85F"
+			tertiary_color = "#FFF9D6"
 		if(4)
-			main_color = "#666655"
-			second_color = "#DDDDEE"
-		if(5)
-			main_color = "#444444"
-			second_color = "#DDDDEE"
+			main_color = "#DB35DE"
+			secondary_color = "#BE3AFE"
+			tertiary_color = "#F5E2EE"
 	human_mob.dna.features["mcolor"] = main_color
-	human_mob.dna.features["mcolor2"] = second_color
-	human_mob.dna.features["mcolor3"] = second_color
+	human_mob.dna.features["mcolor2"] = secondary_color
+	human_mob.dna.features["mcolor3"] = tertiary_color
+
+/datum/species/akula/prepare_human_for_preview(mob/living/carbon/human/akula)
+	var/main_color = "#1CD3E5"
+	var/secondary_color = "#6AF1D6"
+	var/tertiary_color = "#CCF6E2"
+	akula.dna.features["mcolor"] = main_color
+	akula.dna.features["mcolor2"] = secondary_color
+	akula.dna.features["mcolor3"] = tertiary_color
+	akula.dna.mutant_bodyparts["tail"] = list(MUTANT_INDEX_NAME = "Akula", MUTANT_INDEX_COLOR_LIST = list(main_color, secondary_color, tertiary_color))
+	akula.dna.features["legs"] = "Normal Legs"
+	regenerate_organs(akula, src, visual_only = TRUE)
```

Comment:
I'm not sure if that's necessary, but it might be what's causing them to be green in the prefs menu/screenshots tests.
```suggestion
	regenerate_organs(akula, src, visual_only = TRUE)
	akula.update_body(TRUE)
```